### DISCUSSION TITLE
去年的新年新希望有達成嗎？

### DISCUSSION THREAD

**Discussion by Taoyi-Chiang** [1101-GS4719-C/Python-Community-GS4719-C#3692] [General]
每年我都會給自己設定一些目標，有些有達成有些沒有。
我覺得這種目標性的東西在設定的時候先寫下來，等年末再回來看，有時會發現好像一事無成的一年，結果其實還是有滿多成長的。
這種時候就覺得好像也沒有這麼廢。
祝大家新年快樂，新年新希望都可以順利完成！

**Comment by kai1660** [comment]
我達成了，我記得我許願泰文可以一百分，就真的達成了，撒伍迪咖 阿簪陳，希望大家都可以好好學習泰文，已達成新年目標，祝大家都可以選到字喜歡的語文課，撒伍迪咖 阿簪陳。
